Dockerfile

```diff
@@ -12,7 +12,7 @@ FROM alpine:latest
 ARG APP_VERSION=0.0.0
 ENV APP_VERSION=$APP_VERSION
 
-RUN apk add --no-cache ca-certificates
+RUN apk add --no-cache ca-certificates inkscape
```

Comment:
this adds inkscape to the exporter container (I was really happy that there was an api package for it!)

Reply:
Hah, and an alpine package! This was much easier than I was expecting :)